<launch>
<arg name="file_path"   default="$(find map_changer)/config/test1.yaml"/>

<node pkg="map_changer" type="map_changer_node" name="map_changer_node" output="screen">
    <param name="file_path" value="$(arg file_path)"/>
</node>
</launch>